CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/clean/SplitNodesPlugin.java

```diff
@@ -197,20 +197,20 @@ public void edit(final GraphWriteMethods graph, final PluginInteraction interact
         );
         
         //Determine the positions of the selected nodes
-        final List<Integer> selectedVerticies = new ArrayList<>();    
+        final List<Integer> selectedVertexPositions = new ArrayList<>();
         final int graphVertexCount = graph.getVertexCount();
         for (int vertexPosition = 0; vertexPosition < graphVertexCount; vertexPosition++) {
             final int currentVertexId = graph.getVertex(vertexPosition);
             if (graph.getBooleanValue(vertexSelectedAttributeId, currentVertexId)) {
-                selectedVerticies.add(vertexPosition);
+                selectedVertexPositions.add(vertexPosition);
```

Comment:
I wonder if the optimal fix here actually just stores the vertex id here instead of the position. Looking at the subsequent for loop code (iterating over the list), the positions are only then used for:
1) Getting the vertex id
2) Incorrectly being used where the vertex id should've been used

So really parsing the vertex id later on as you've done fixes 2) and by then storing a list of ids instead of positions, there is also no need for 1) (line 221)